.github/workflows/docker.yml

```diff
@@ -0,0 +1,45 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - published
+
+jobs:
+  containerize:
+    strategy:
+      matrix:
+        app:
+          - Api
+          # - AdminConsole
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Install Docker Buildx
+        uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226 # 3.0.0
+
+      - name: Login to GitHub Container Registry
+        run: >
+          echo ${{ secrets.GITHUB_TOKEN }} |
+          docker login ghcr.io --username ${{ github.actor }} --password-stdin
+
+      - name: Build & push image
+        run: >
+          docker buildx build
+          --file ${{ matrix.app }}.dockerfile
+          --platform linux/amd64,linux/arm64
+          --push
+          --tag ghcr.io/passwordless/api-test-server:latest
```

Comment:
This or `bitwarden/passwordless-api-test-server`, @abergs? It needs to highlight that it's `api`

Reply:
Let's not use /bitwarden until it's ready. Passwordless/passwordless-test-api ? (And then we will just switch to bitwarden/ later)